enhancements/baremetal/baremetal-keepalived-unicast.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Baremetal-Keepalived-Unicast
+authors:
+  - "@bcrochet"
+  - "@yboaron"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-02-21
+last-updated: 2020-02-21
+status: implementable
+---
+
+# baremetal-keepalived-unicast
```

Comment:
There's a section in the template about upgrade/downgrade impact, as well as version skew handling.  I was expecting to see those topics explored in this doc.  Can you take a look at adding those?

Reply:
Sure, good point. 
I'll add that to the PR.

Reply:
Done